modules/oxide/src/less/theme/components/editor/editor.less

```diff
@@ -9,6 +9,9 @@
 @editor-bottom-header-sticky-box-shadow: @editor-header-sticky-box-shadow;
 @editor-header-sticky-background-color: if((alpha(@toolbar-background-color) > 0), transparent, @background-color); // If @toolbar-background-color is set to transparent, then set the sticky toolbar background color to be another color to not be transparent over content.
 
+// private variables
+@_editor-bottom-header-separator: if(alpha(@menubar-separator-color) > 0, @menubar-separator-color, @tinymce-separator-color); // Use @menubar-separator-color if it is set, otherwise use default separator color
```

Comment:
consider point that it is color: `@_editor-bottom-header-separator-color`

Reply:
Fair point, thanks!